cfn-resources/private-endpoint/cmd/constants/event_status.go

```diff
@@ -0,0 +1,31 @@
+package constants
+
+import (
+	"fmt"
+)
+
+type EventStatus string
+
+const (
+	CreationInit                   EventStatus = "CREATION_INIT"
+	CreatingPrivateEndpointService EventStatus = "CREATING_PRIVATE_ENDPOINT_SERVICE"
+	CreatingPrivateEndpoint        EventStatus = "CREATING_PRIVATE_ENDPOINT"
+)
+
```

Comment:
the ParseEventStatus is used to get the Constant from the string in the callback, i also added an extra layer of validation in case something brokes in the process and we have some unexpected value in the Callback status, i can throw an specific error.  you can check this here https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/144/files/30becdfa4efb0b5698cc0042487fddcd2e6f2b25#diff-e3aabda7072d2f0e3ba88e71033c379d59eaa57c3d7e92fdd216cdf86da03f08R273